packages/api/cms-api/src/file-uploads/file-uploads.service.ts

```diff
@@ -90,4 +90,11 @@ export class FileUploadsService {
 
         return ["/file-uploads", hash, file.id, timeout, resizeWidth, filename].join("/");
     }
+
+    async delete(file: FileUpload): Promise<void> {
+        if (await this.blobStorageBackendService.fileExists(this.config.directory, file.name)) {
+            await this.blobStorageBackendService.removeFile(this.config.directory, file.name);
+            await this.repository.nativeDelete({ id: file.id });
+        }
+    }
```

Comment:
- why nativeDelete? (vs. "normal" remove())
- why only delete in databse if fileExists?

Reply:
98d28bd5507b33b0d062089668aa008d9953f066